ccpp/data/GFS_typedefs.meta

```diff
@@ -6375,6 +6375,13 @@
   dimensions = (horizontal_loop_extent)
   type = real
   kind = kind_phys
+[paha]
```

Comment:
This array is now only allocated if the LSM is Noah MP. You need to add a line `active = (control_for_land_surface_scheme == identifier_for_noahmp_land_surface_scheme)` for each of those variables that are allocated conditionally. Please see the existing entry for variable `snowxy` for an example.

Reply:
made the modification

Reply:
Thanks, that looks great!